### PR TITLE
 Add additional fields & Update CHANGELOG

### PR DESCRIPTION
**Pull request by pavansoundara**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog for package power_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
```
Comment (cjds) [inline CHANGELOG.rst]: 0.3.1 we'll tag it afterwards

**Review by cjds — CHANGES_REQUESTED** [review]
While we're doing this could you follow package format 2. Reference https://github.com/fetchrobotics/fetch_msgs/pull/14/files

**Review by cjds — APPROVED** [review]
LGTM. @moriarty what are the next steps for this?

File: CHANGELOG.rst (hunk 1 above)
Comment (moriarty) [reply CHANGELOG.rst]: this breaks API doesn't it?
So it should probably be a 0.4.0 or 1.0.0

I'll use `catkin_prepare_release --bump` to do all this for us though.

**Review by moriarty — COMMENTED** [review]
The next step is I add it to my list of things to bloom, tomorrow or Friday

**Comment by moriarty** [comment]
@cjds I think this will mean we need to recompile and re-release the fetch_binary_drivers

**Comment by moriarty** [comment]
@cjds I'm wondering if we can move towards the standard BatteryState msg, it was added in 2015 and has everything we need. The N-Turtle and ROS2 versions also include fields for temperature.

https://github.com/ros/common_msgs/blob/jade-devel/sensor_msgs/msg/BatteryState.msg

**Comment by cjds** [comment]
Yeah we should be able to

File: msg/BatteryState.msg [hunk 2]
```diff
@@ -14,3 +14,19 @@ duration remaining_time
 
 # The list of errors associated wtih the charger
 string[] errors
```
Comment (moriarty) [inline msg/BatteryState.msg]: I don't see `string[] errors` in the the`fetch_msgs_internal` version of this

**Comment by moriarty** [comment]
I've rebased and edited the original branch, synced it with the internal version and force pushed to @pavansoundara's branch...

**Comment by moriarty** [comment]
https://github.com/ros/rosdistro/pull/22107

**Pull request merged**